win32k-tracing.js

```diff
@@ -1155,11 +1155,24 @@ function _executeCommand(cmd) {
     host.namespace.Debugger.Utility.Control.ExecuteCommand(cmd);
 }
 
+function decimalBufferToString(buf) {
+    var str = "";
+
+    for (let i = 0; i < 512; ++i) {
```

Comment:
it looks like `CommandLine` has a `Length` attribute, in addition to the `Buffer` one, can we use that?

Reply:
I looked back into it more in-depth and as far as I can tell the length given by `CommandLine` is always larger than what is actually in the buffer, but it's definitely a better guess than a hard-coded 512.